lib/src/config/config_file/client.example.toml

```diff
@@ -26,7 +26,10 @@ listen_addresses = [
 
 # Seed nodes to connect to when the node starts up. Multiple addresses can be specified.
 seed_nodes = [
-  { address = "/dns4/seed1.pos.nimiq-testnet.com/tcp/8443/wss" },
```

Comment:
Please state the testnet seed node entry in the doc comment, so people can still find it easily.